packages/react/src/components/Tag/Tag-test.js

```diff
@@ -17,6 +20,27 @@ describe('Tag', () => {
       await expect(container).toHaveNoAxeViolations();
     });
 
+    it('should have no Axe violations DismissibleTag', async () => {
+      const { container } = render(
+        <DismissibleTag type="red">test-tag</DismissibleTag>
+      );
+      await expect(container).toHaveNoAxeViolations();
+    });
+
+    it('should have no Axe violations OperationalTag', async () => {
+      const { container } = render(
+        <OperationalTag type="red">test-tag</OperationalTag>
+      );
+      await expect(container).toHaveNoAxeViolations();
+    });
+
+    it('should have no Axe violations SelectableTag', async () => {
+      const { container } = render(
+        <SelectableTag type="red">test-tag</SelectableTag>
+      );
+      await expect(container).toHaveNoAxeViolations();
+    });
```

Comment:
Do we need these tests anymore since we now run the a11y-checker in the e2e AVT tests?